client/state/jetpack/credentials/schema.js

```diff
@@ -0,0 +1,14 @@
+export const itemsSchema = {
+	type: 'object',
+	items: {
+		type: 'object',
+		properties: {
+			abspath: { type: 'string' },
+			host: { type: 'string' },
+			port: { type: 'number' },
+			protocol: { type: 'string' },
+			pass: { type: 'bool' },
```

Comment:
Wow, in the end this PR likely is causing those e2e tests to fail. This line breaks deserialization, meaning any e2e tests that (rightly or wrongly) rely on persisted state may fail.

Error:
```
calypso:state failed to load initial redux-store state +75ms TypeError: types[(t || "any")] is not a function
    at eval (VM8962 index.js:183)
    at Array.map (<anonymous>)
    at visit (VM8962 index.js:182)
    at eval (VM8962 index.js:532)
    at Array.forEach (<anonymous>)
    at visit (VM8962 index.js:529)
    at visit (VM8962 index.js:363)
    at compile (VM8962 index.js:546)
    at module.exports (VM8962 index.js:575)
    at isValidStateWithSchema (VM8953 utils.js:71)
```

Fix:
`pass: { type: 'boolean' },`